AngouriMath/Functions/NumbersExtensions.cs

```diff
@@ -149,6 +155,10 @@ public static EDecimal Sin(this EDecimal x, EContext context)
             return CalculateSinFromCos(x, cos, consts, context);
         }
 
+        public static EDecimal Signum(this EDecimal x, EContext _)
+            => x.Sign;
```

Comment:
This extension method should be removed, use the property directly

Reply:
It's fine for consistency.